template/node-armhf/template.yml

```diff
@@ -0,0 +1,2 @@
+language: node-armhf
+fprocess: node index.js
```

Comment:
Does the node template contain a full template or does it still only have the Dockerfile patch? cc @rgee0